### ISSUE TITLE
Rgbd_segmentation

### ISSUE THREAD

**Issue by jonahgeladze** [TUI-NICR/ESANet#7]
I'm using rgbd segmentation pretrained model. Can you tell me, how to get a depth image of room? 

Thanks, in advance.

**Comment by mona0809** [comment]
sorry, but what do you mean? The model takes RGB and depth images as input (e.g. from a Kinect camera) and outputs the segmentation result.

**Comment by jonahgeladze** [comment]
@mona0809 I'm using sunrgbd model. I want to test this model on my samples(room images). So from 2 inputs, I have only room image (and not depth image). Is there any chance to transform my room image in depth image? (So I would have 2 input parameters: room image and depth image) and I'd be able to run my inputs on this model.

**Comment by mona0809** [comment]
No, it is not possible to reconstruct a depth image given an RGB image. (There are approaches for depth estimation, but that is another area of research.) If you only have the RGB image, I suggest you use a model that requires only the RGB image as input.
